Update terms used for team accounts/admins (#71)
HamStudy/examtools-docs: content/docs/cve/cveTeams.md

- **Team Name** - the team’s name or group associated with the team account
- **Contact E-mail Address** - a valid email address associated with the administrator (owner) of the team
- **VEC** (Which of your VEC credentials should this team be allowed to use to schedule exams?) - Select an available VEC from the drop-down menu.  The individual setting up the account will have the option of selecting from the VECs for which they are a session manager/team lead/liaison in ExamTools.  Select all that apply to this account. 
- When done, click **Save** to exit.

## Team Accounts : To edit or view team accounts

 * Within **Team Accounts**, locate the account you wish to view and click on the **Manage Account** button which will allow you to view all the fields and setting for the account.
 *  **Team Identifier:**  - Used by the applicants to login and join the exam; also used in permalinks for the session and the team on HamStudy.
 *  **Team Name:** - Displays along with the team identifier in the session listing on HamStudy.

>  *  **Contact E-mail Addresses:** - Should contain a valid email addresses for the administrators of the account.

Reviewer: @taxilian The should not be pluralized unless it allows entering multiple email addresses.
```suggestion
 *  **Contact E-mail Address:** - Should contain a valid email address for the administrator(s) of the account.
```

Reviewer: I see you can add additional addresses *after* creating the team on the "manage" page. Perhaps the initial "New Team Account" modal dialog should allow adding multiple email addresses as well or change the wording.

Reviewer: @hparkersd (these comments are for Richard to change the ET UI)

Reviewer: Let's wait to see if @taxilian decides to change the ET code or not.

 *  **Team Website:** - Enter the url (i.e. http://…) for the website for the club or group associated with this team account. Leave blank if you do not have one.
 *  **My team permissions:** - The personal permissions settings for the team account you are viewing. (not editable and set by the owner/co-owner of the session)
 *  **Delegated Privileges:** (In order to list sessions for a given VEC one or more VEs who are at least co-owners must delegate their team lead permissions to the team. Note that sessions must still have a valid team lead assigned from the team, but anyone with Scheduling permissions may schedule sessions for any VEC for which the team has delegated permissions.) - Only those with session manager permissions in ExamTools will see VECs in the Delegate Privileges list.  Select the VEC for which you would like the team schedulers to be able to create sessions for.  If selected, the VEC will display below the feature.
 

>  ## Add, Edit, and Delete Administers

Reviewer: ```suggestion
 ## Add, Edit, and Delete Administrators
```

 
 *  **Adminstrators:** - A list of those VEs who are part of your team management.  This list does not include everyone of your VEs, but rather those few who assist with the scheduling, coordinating and running of the sessions.

>  *  Click on the **+ INVITE** next to **Administrator:** to add administrators whoc will need to accept your invitation. 

Reviewer: ```suggestion
 *  Click on the **+ INVITE** next to **Administrator:** to add administrators who will need to accept your invitation. 
```
Per your "tired" comment in Discord, maybe don't request a review until you've taken a second look and fixed typos.
